### PR TITLE
Embed Widget: Use 'canonical' as referrer, if present

### PR DESCRIPTION
**Pull request by lukasender**

### CODE REVIEW

File: src/embed.js [hunk 1]
```diff
@@ -19,7 +19,16 @@ const fetchHeight = (child) => {
...
+  const linkElements = document.getElementsByTagName('link');
+  const linksArray = Array.prototype.slice.call(linkElements);
+  // get the canonical URL
+  let referrer = null;
+  linksArray.forEach((linkElement) => {
+    if (linkElement.rel === 'canonical' && !referrer) {
+      referrer = linkElement.href;
+    }
+  });
+  const url = getPetitionEmbedUrl(id, referrer);
```
Comment (lusbuab) [inline src/embed.js]: is there a reason to not use `querySelector`: 

```javascript
const canonicalLink = document.querySelector('link[rel="canonical"]')
// canonicalLink is now an element or null
```

Comment (lusbuab) [reply src/embed.js]: http://caniuse.com/#search=querySelector

Comment (lukasender) [reply src/embed.js]: i don't know. don't we have to support older browsers?

Comment (lukasender) [reply src/embed.js]: if not, i'd happily use that one instead

Comment (lusbuab) [reply src/embed.js]: even IE9 supports it: http://caniuse.com/#search=querySelector (click "Show All")

Comment (lukasender) [reply src/embed.js]: very well then :D

**Review by lusbuab — APPROVED** [review]

**Pull request merged**